core/common/src/main/java/alluxio/underfs/BaseUnderFileSystem.java

```diff
@@ -77,6 +79,11 @@ public boolean exists(String path) throws IOException {
     return isFile(path) || isDirectory(path);
   }
 
+  @Override
+  public AccessControlList getAcl(String path) throws UnimplementedException, IOException {
+    throw new UnimplementedException("ACL is not supported");
```

Comment:
Instead of using an exception, could we return some default value?

Reply:
null is returned